src/components/instant-buy/processing.tsx

```diff
@@ -34,10 +34,16 @@ const Processing = (props: Props) => {
 				return
 			}
 			const { data } = res as PaymentStatusProps
-			if (
-				data.paymentState === "PAID" ||
-				data.paymentState === "ALREADY_PROCESSED"
-			) {
+
+			let paymentState = "INITIATED"
+
+			if ((data.paymentState as string) === "PAID") {
```

Comment:
Use PROCESSING and SUCCESSFUL directly. PAID and ALREADY_PROCESSED will never come from the backend going forward.